hack/install.sh

```diff
@@ -51,6 +51,23 @@ echo_docker_as_nonroot() {
 	EOF
 }
 
+# Check upstream release in forked Linux distros
+check_forked() {
+	if [ "$lsb_dist" = "$1" ]; then
```

Comment:
We'll also want to check for linuxmint, cumulus linux, kali, etc. here. Right now, if we want to check multiple distros we have to call check_forked multiple times. I'm not sure if that's the cleanest way? Other thoughts I have would be to either hardcode the list in this if check or perhaps even better if it works across distros would be to remove the if check entirely and always find lsd_dist and dist_version the way that's being done here